sdk/communication/Azure.Communication.CallingServer/tests/Infrastructure/CallingServerLiveTestBase.cs

```diff
@@ -115,6 +117,35 @@ protected CallingServerClient CreateInstrumentedCallingServerClient()
             return InstrumentClient(callingServerClient);
         }
 
+        /// <summary>
+        /// Creates a <see cref="CallingServerClient" />.
+        /// </summary>
+        /// <returns>The instrumented <see cref="CallingServerClient"/>.</returns>
+        protected CallingServerClient CreateInstrumentedCallingServerClientWithToken()
+        {
+            Uri endpoint = TestEnvironment.LiveTestStaticEndpoint;
+            TokenCredential tokenCredential;
+
+            if (Mode == RecordedTestMode.Playback)
+            {
+                tokenCredential = new MockCredential();
+            }
+            else
+            {
+                #region Snippet:Azure_Communication_CallingServer_Tests_Samples_CreateCallingServerClientWithToken
+                //@@ string endpoint = "<endpoint_url>";
+                //@@ string userAssignedClientId = "<your managed identity client Id>";
+                //@@ var tokenCredential = new DefaultAzureCredential(new DefaultAzureCredentialOptions { ManagedIdentityClientId = userAssignedClientId });
+                /*@@*/
+                tokenCredential = new DefaultAzureCredential();
```

Comment:
line 139 and 140 should be same line. The readme snippet will also need to be updated.

Reply:
Updated it here